qf_lib/tests/integration_tests/backtesting/test_backtester.py

```diff
@@ -53,26 +52,25 @@ def calculate_and_place_orders(self):
 
     def calculate_signals(self):
         if not self.invested:
-            orders = self.order_factory.percent_orders({self.MICROSOFT_TICKER: 1.0}, MarketOrder(),
-                                                       TimeInForce.GTC)
+            orders = self.order_factory.percent_orders({self.AAA_TICKER: 1.0}, MarketOrder(), TimeInForce.GTC)
             self.broker.place_orders(orders)
             self.invested = True
 
 
 class TestBacktester(TestCase):
     def setUp(self):
         try:
-            self.data_provider = get_data_provider()
+            self.data_provider = daily_data_provider
```

Comment:
This class is used by our internal library. It is crucial for us to not change its implementation - please revert to use the BloombergDataProvider both here and in the BuyAndHoldStrategy

Reply:
ok, I'm rolling back this file